packages/frontend/web/components/ArticleBody.tsx

```diff
@@ -361,14 +357,36 @@ const bylineLink = css`
 `;
 
 const ageWarning = css`
-    ${textSans(1)};
+    ${textSans(5)};
+    color: ${palette.neutral[7]};
+    background-color: ${palette.highlight.main};
     display: inline-block;
-    margin-bottom: 12px;
-    width: 100%;
+    margin-bottom: 6px;
 
-    ${leftCol} {
-        margin-top: 6px;
+    > strong {
+        font-weight: bold;
     }
+
+    padding: 6px 10px;
+    margin-top: 6px;
+    margin-left: -10px;
+
+    ${mobileLandscape} {
+        padding-left: 12px;
+    }
+
+    ${tablet} {
+        margin-left: -20px;
+    }
+
+    ${leftCol} {
+        margin-left: -10px;
+        margin-top: -6px;
+        padding-left: 10px;
+`;
```

Comment:
can we extract these negative margins into a function in the container that applies the paddings they are counter acting? that way we keep them in the same place

Reply:
I'd love to do this but there's a lot of (unnecessary) complexity around the way the article and these grid sections are spaced. Can we accept this as tech debt for now, and I'll add it as a general issue to our Trello board?

Reply:
let's!